tests/Unit/OutputFormatTest.php

```diff
@@ -1025,4 +1035,133 @@ public function createCompactReturnsInstanceWithRenderCommentsDisabled(): void
 
         self::assertFalse($newInstance->getRenderComments());
     }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsNewOutputFormatInstance(): void
+    {
+        self::assertInstanceOf(OutputFormat::class, OutputFormat::createPretty());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyCalledTwoTimesReturnsDifferentInstances(): void
```

Comment:
I'm thinking (and agreeing) "TwoTimes" is better than "Twice" for non-native-English speaking coders.